recipes/default.rb

```diff
@@ -286,8 +278,6 @@
          :dela_ip => dela_ip,
          :krb_ldap_auth => node['ldap']['enabled'].to_s == "true" || node['kerberos']['enabled'].to_s == "true",
          :featurestore_jdbc_url => featurestore_jdbc_url,
-         :registry_host => registry_host,
-         :hops_version => get_hops_version(node['hops']['version'])
```

Comment:
@berthoug why is the hops_version removed?

Reply:
probably a mistake when fixing conflicts. I'm fixing it.